src/Panel/Document.php

```diff
@@ -80,27 +81,57 @@ public static function assets(): array
                 ]
             ]),
             'js' => [
-                'vendor'       => $url . '/js/vendor.js',
-                'pluginloader' => $url . '/js/plugins.js',
-                'plugins'      => $plugins->url('js'),
-                'custom'       => static::customJs(),
-                'index'        => $url . '/js/index.js',
+                'vendor'       => [
+                    'nonce' => $nonce,
+                    'src'   => $url . '/js/vendor.js',
+                    'type'  => 'module'
+                ],
+                'pluginloader' => [
+                    'nonce' => $nonce,
+                    'src'   => $url . '/js/plugins.js',
+                    'type'  => 'module'
+                ],
+                'plugins'      => [
+                    'nonce' => $nonce,
+                    'src'   => $plugins->url('js'),
+                    'defer' => true
```

Comment:
What happens when you still add type module?

Reply:
Console:
> Uncaught TypeError: Cannot read properties of undefined (reading 'TYPED_ARRAY_SUPPORT')

Panel:
> The section type "pagetable" is not valid